Source/Scene/WebMapServiceImageryProvider.js

```diff
@@ -75,14 +83,8 @@ define([
      *        {@link WebMapServiceImageryProvider#pickFeatures} will immediately return undefined (indicating no pickable features)
      *        without communicating with the server.  Set this property to false if you know your WMS server does not support
      *        GetFeatureInfo or if you don't want this provider's features to be pickable.
-     * @param {Boolean} [options.getFeatureInfoAsGeoJson=true] true if {@link WebMapServiceImageryProvider#pickFeatures} should
```

Comment:
Ah, nevermind, it is taken care of.  Can you submit an issue for 1.13 and add a label if there isn't already one.